src/classes/MapHierarchalPractice.cls

```diff
@@ -0,0 +1,157 @@
+/*
+CountryCode 
+    -country
+        -state
+            -district
+*/
+public with sharing class MapHierarchalPractice {
```

Comment:
Add description above the class

/** 
  *    Description     :    Purpose and fucntionality of clss
  *
  *    Created By      :   Dev NAme    
  *
  *    Created Date    :    
  * 
  *    Revision Logs   :    V_1.0 - Created
  *                      
  **/